manubot/process/header-includes-template.html

```diff
@@ -0,0 +1,75 @@
+<!--
+Manubot generated metadata rendered from header-includes-template.html.
+Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
+-->
+<meta name="dc.format" content="text/html" />
+{% if pandoc.title is defined -%}
+<meta name="dc.title" content="{{ pandoc.title|escape }}" />
+<meta name="citation_title" content="{{ pandoc.title|escape }}" />
+<meta property="og:title" content="{{ pandoc.title|escape }}" />
+<meta property="twitter:title" content="{{ pandoc.title|escape }}" />
+{% endif -%}
+{% if pandoc['date-meta'] is defined -%}
+<meta name="dc.date" content="{{ pandoc['date-meta'] }}" />
+<meta name="citation_publication_date" content="{{ pandoc['date-meta'] }}" />
+{% endif -%}
+{% if pandoc.lang is defined -%}
+<meta name="dc.language" content="{{ pandoc.lang }}" />
+<meta name="citation_language" content="{{ pandoc.lang }}" />
+{% endif -%}
+<meta name="dc.relation.ispartof" content="Manubot Preprint" />
+<meta name="dc.publisher" content="Manubot" />
+<meta name="citation_journal_title" content="Manubot Preprint" />
+<meta name="citation_technical_report_institution" content="Manubot" />
+{% for author in manubot.authors|default([]) if author.name is defined -%}
+<meta name="citation_author" content="{{ author.name|escape }}" />
+{% if author.affiliations is defined -%}
+{% for affiliation in author.affiliations -%}
+<meta name="citation_author_institution" content="{{ affiliation|escape }}" />
+{% endfor -%}
+{% endif -%}
+{% if author.orcid is defined -%}
+<meta name="citation_author_orcid" content="{{ author.orcid }}" />
+{% endif -%}
+{% if author.twitter is defined -%}
+<meta name="twitter:creator" content="@{{ author.twitter }}" />
```

Comment:
Can this be specified multiple types?  https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started refers to "individual user".

Reply:
The docs seem not to consider the situation of multiple authors:

> the most appropriate Twitter account should be used to provide the best context for the user. For example, nytimes.com may set the the website attribution to “`@nytimes`” for front page articles, and “`@NYTArts`” for articles in the Arts & Entertainment section.

I think you are right that they don't support multiple `twitter:creator` properties. I am guessing applications that expect zero or one `twitter:creator` will be able to handle the two+ case by ignoring all but the first or last `twitter:creator`. I think we might as well put every creator in, since anything else seems incomplete.